modules/install/pages/upgrade-to-ipv6.adoc

```diff
@@ -0,0 +1,42 @@
+= Upgrading to IPv6
+
+[abstract]
+A cluster that is upgraded to the latest version of Couchbase Server can be specified to use the IPv6 address family.
+
+== Clusters Currently Using IPv4
+
+A pre-6.5 Couchbase Server-cluster that is currently using the IPv4 address family can be upgraded to run 6.5 _and_ to use IPv6.
+The recommended upgrade path is the following:
+
+. Upgrade the cluster to Couchbase Server Enterprise Edition, Version 6.5, continuing to use IPv4.
+For instructions, see xref:install:upgrade-strategies.adoc[Upgrade Options].
+
+. Change the 6.5 cluster's address family from IPv4 to IPv6.
+For instructions, see xref:manage:manage-nodes/manage-address-families.adoc[Manage Address Families].
+
+== Clusters Currently Using IPv6
+
+To perform offline upgrade for a running 5.5.0, 5.5.1, 5.5.2, or 6.0.0 Couchbase Server-cluster that is currently using IPv6, to Couchbase Server Enterprise Edition 6.5 &#8212; and have it continue to use IPv6 &#8212; special configuration steps are required.
+
+Proceed as follows:
+
+. Upgrade the cluster to Couchbase Server Enterprise Edition, Version 6.5.
+For instructions, see xref:install:upgrade-offline.adoc[Performing the Offline Upgrade].
+
+. Stop Couchbase Server.
+For instructions, see xref:install:startup-shutdown.adoc[Start and Stop Couchbase Server].
+
+. As either _root_ or _couchbase user_, enter the following command:
++
+----
+if [ -e /opt/couchbase/var/lib/couchbase/config/dist_cfg ] && [ ! -s /opt/couchbase/var/lib/couchbase/config/dist_cfg ]; then
+  echo "[{preferred_external_proto,inet6_tcp_dist}, {preferred_local_proto, inet6_tcp_dist}]." > /opt/couchbase/var/lib/couchbase/config/dist_cfg;
+fi
+----
+
+. Restart Couchbase Server.
+
+This concludes the upgrade-process.
+
+Note that the above configuration-steps are only required for upgrade to 6.5 of clusters already using IPv6.
```

Comment:
for only "5.5.0, 5.5.1, 5.5.2, or 6.0.0" versions
5.5.3 and 6.0.1 and higher do not require those extra steps even if they use IPv6.